native/android/app/build.gradle

```diff
@@ -162,6 +162,11 @@ dependencies {
     // compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.android.support:appcompat-v7:25.3.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
+    compile "org.mozilla.components:fxa:0.16"
+}
+
+repositories {
+    mavenLocal()
```

Comment:
FYI: 0.16 has been released yesterday and can be pulled from jcenter now. :)

Reply:
I guess there is no issue keeping this setting. I think if it cannot find it locally it will go to jcenter